apps/frontend/src/views/Results.vue

```diff
@@ -282,6 +285,26 @@ export default class Results extends Vue {
   /** Determines if we should make the search bar colapseable */
   show_search_mobile: boolean = false;
 
+  /**
+   * On mount, check if we have been passed an evaluation
+   * to load from Vue router
+   */
+  async mounted() {
+    if(this.$route.params.id){
+      if(this.$route.params.id === 'results') {
+        return
+      }
+      await EvaluationModule.getAllEvaluations();
+      const evaluationsToLoadArray: IEvaluation[] = await EvaluationModule.findEvaluationsByIds(this.$route.params.id.split(','));
+      if (evaluationsToLoadArray.length !== 0) {
+        EvaluationModule.load_results(evaluationsToLoadArray)
+      } else {
+        SnackbarModule.failure(`Heimdall was passed the following evaluations to open, but couldn't find any of them: ${this.$route.params.id}`);
+        this.$router.push('/results');
+      }
```

Comment:
Instead of doing the check against all of the evaluations, if the change below is made you can just go ahead and make a call to `EvaluationModule.load_results(this.$route.params.id.split(','))`.

Reply:
load_results takes a list of IEvaluations, not just IDs, so unfortunately we have to find each one that's passed before loading

Reply:
That was my reason for suggesting the change of the type of `load_results` from a list of evaluations to just an ID. It will require a few other small changes to `load_results` but all of the data should be available as part of the response from ```axios.get<IEvaluation>(`/evaluations/${evaluation.id}`)```